docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sumo-logic-sample-data-source.md

```diff
@@ -0,0 +1,91 @@
+---
+id: sumo-logic-sample-data-source
+title: Sumo Logic Sample Data Source
+sidebar_label: Sumo Logic Sample Data
+keywords:
+  - sumo-logic-sample-data
+  - cloud-to-cloud
+description: Learn how to collect audit reporting events from Sumo Logic Sample Data platform.
+---
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/files/c2c/sumo-logic-sample-data/example.json';
+import MyComponentSource from '!!raw-loader!/files/c2c/sumo-logic-sample-data/example.json';
+import TerraformExample from '!!raw-loader!/files/c2c/sumo-logic-sample-data/example.tf';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+<img src={useBaseUrl('img/send-data/sumo-logic-sample-data.png')} alt="thumbnail icon" width="55"/>
+
+Sumo Logic Sample Data source is designed for ingesting logs and metrics into Sumo Logic for the OpenTelemetry demo application. The data source is engineered to provide a continuous stream of data that simulates a specific scenario, highlighting latency spikes across various services.
+
+## Data collected
+
+| Name | Description |
+| :--- | :--- |
+| Initial Data             | The source ingests data from `6 hours in the past` and `6 hours into the future`. |
+| Regular Ingestion        | Every 6 hours, the source ingests an additional `6 hours of future-dated data` to ensure dashboards are always populated. |
```

Comment:
```suggestion
| Initial Data             | The source ingests data from 6 hours in the past and 6 hours into the future. |
| Regular Ingestion        | Every 6 hours, the source ingests an additional 6 hours of future-dated data to ensure dashboards are always populated. |
```

Reply:
@ishaanahuja29, we use single backticks to show inline code (see the [style guide](https://help.sumologic.com/docs/contributing/style-guide/#code-inline)), not for emphasis.